internal/logger/logger.go

```diff
@@ -101,50 +103,58 @@ func Close() {
 	}
 }
 
-// NewNotice returns a new logger for logging notice with given prefix to
-// the log file or the status writer which forwards the notices to the invoker
-// from the daemon.
-func NewNotice(prefix string) *log.Logger {
-	return defaultLoggerFactory.newLogger("NOTICE", prefix)
-}
-
 // NewDebug returns a new logger for logging debug messages with given prefix
 // to the log file or stdout.
 func NewDebug(prefix string) *log.Logger {
-	return defaultLoggerFactory.newLogger("DEBUG", prefix)
+	return defaultLoggerFactory.NewLogger(LevelDebug, prefix)
 }
 
 // NewInfo returns a new logger for logging info with given prefix to the log
 // file or stdout.
 func NewInfo(prefix string) *log.Logger {
-	return defaultLoggerFactory.newLogger("INFO", prefix)
+	return defaultLoggerFactory.NewLogger(LevelInfo, prefix)
 }
 
 // NewError returns a new logger for logging errors with given prefix to the log
 // file or stderr.
 func NewError(prefix string) *log.Logger {
-	return defaultLoggerFactory.newLogger("ERROR", prefix)
+	return defaultLoggerFactory.NewLogger(LevelError, prefix)
 }
 
 // Errorf calls the default error logger to print the message using Printf.
```

Comment:
We can correct these comments, as there will be only logger with different severities.

Reply:
I need to clean up these methods after all the slog PRs are merged. Adding a TODO instead.